### PR TITLE
[HDInsight] - GA release

### PR DESCRIPTION
**Pull request by idear1203**
* resize: Made --workernode-count/-c required
* GA release

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -34,6 +34,19 @@ Release History
...
+
+* Expanded `--json-file` capabilities of `az batch pool create` to allow for specifying MountConfigurations for file system mounts(see https://docs.microsoft.com/en-us/rest/api/batchservice/pool/add#request-body for structure)
+* Expanded `--json-file` capabilities of `az batch pool create` with the optional property publicIPs on NetworkConfiguration. This allows specifying publicIPs to be used when deploying pools (see https://docs.microsoft.com/en-us/rest/api/batchservice/pool/add#request-body for structure)
+* Expanded `--image` capabilities to support Shared Image Galleries images. Similar to the commands support for Managed Images, to use a Shared Image Gallery image simply use the ARM ID as the value to the argument.
+* [BREAKING] When not specified, the default value for `--start-task-wait-for-success` on `az batch pool create` is now true (was false).
+* [BREAKING] The default value for Scope on AutoUserSpecification is now always Pool (was Task on Windows nodes, Pool on Linux nodes). This argument is not exposed via the commandline, but can be set in the `--json-file` arguments.
+
+**HDInsight**
+
+* resize: Made --workernode-count/-c required
```
Comment (yugangw-msft) [inline src/azure-cli/HISTORY.rst]: Please rebase to get rid of the change logs from `Batch`

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -38,7 +38,12 @@ Release History
 
 * az webapp up detects env folder and removes it from compressed file used for deployment
 
-**Keyvault**
+**HDInsight**
+
+* resize: Made --workernode-count/-c required
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: The following description may be better:
`az hdinsight resize`: Make parameter `--workernode-count/-c` required

Comment (idear1203) [reply src/azure-cli/HISTORY.rst]: Fixed

**Comment by Juliehzl** [comment]
Looks good to me in general.  Can you consider my suggestion for description?

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (idear1203) [reply src/azure-cli/HISTORY.rst]: Fixed

**Pull request merged**